packages/@okta/vuepress-site/docs/reference/api-overview/index.md

```diff
@@ -236,7 +236,7 @@ The Okta API had incorporated [JSON Hypertext Application Language](http://tools
 
 ## Resources
```

Comment:
I'm not sure what the point of this section is. It seems to be defining "Resources" which is not a term we use anywhere else in the docs. Maybe this section needs to be renamed "Objects" and talk about those?